server/src/main/java/org/opensearch/cluster/routing/allocation/AllocationService.java

```diff
@@ -565,8 +567,15 @@ private void reroute(RoutingAllocation allocation) {
         long rerouteStartTimeNS = System.nanoTime();
         removeDelayMarkers(allocation);
 
+        long startTime = System.nanoTime();
         allocateExistingUnassignedShards(allocation);  // try to allocate existing shard copies first
+        logger.info("Completing allocate unassigned, elapsed time: [{}]", TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime));
```

Comment:
can these be debug logs

Reply:
Yes, will update log levels before closing this PR